share/we-lang/we-lang.go

```diff
@@ -1169,11 +1169,11 @@ func main() {
 	out := formatCond(make([]string, 5), r.Data.Cur[0], true)
 	for _, val := range out {
         if config.RightToLeft {
-            space := strings.Repeat(" ", 93)
-            fmt.Fprintln(stdout, space, val)
+            fmt.Fprint(stdout, strings.Repeat(" ", 94))
```

Comment:
RTL was/is also aligned:
![Screenshot from 2020-06-27 21-38-08](https://user-images.githubusercontent.com/697976/85931037-73401580-b8c1-11ea-832c-6c7d648f2723.png)